src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -50,17 +52,17 @@ public void setUp() {
         logic = new LogicManager(model, storage);
     }
 
-    @Test
-    public void execute_invalidCommandFormat_throwsParseException() {
-        String invalidCommand = "uicfhmowqewca";
-        assertParseException(invalidCommand, MESSAGE_UNKNOWN_COMMAND);
-    }
+    // @Test
+    // public void execute_invalidCommandFormat_throwsParseException() {
```

Comment:
Thanks for commenting out the tests. Note that you can use 
```
/* 
sample code 
*/
```
to comment out huge chunks of code, rather than use // line by line.

Reply:
Good suggestion!
The reason why I didn't use that because I was afraid it would fail the checkstyle because of the use of wildcards.